user_docs/en/changes.md

```diff
@@ -71,6 +71,7 @@ A warning message will inform you if you try writing to a non-empty directory. (
   * NVDA will correctly announce radio and checkbox menu items when first entering sub-menus in Google Chrome and Mozilla Firefox. (#14550)
   * NVDA's browse mode find functionality is now more accurate when the page contains emojis. (#16317, @LeonarddeR)
   * In Mozilla Firefox, NVDA now correctly reports the current character, word and line when the cursor is at the insertion point at the end of a line. (#3156, @jcsteh)
+  * No longer cause Google Chrome to crash when closing a document or exiting Chrome. (#16893)
```

Comment:
@michaelDCurran or @seanbudd, I am reviewing the translation of this item.

Translating this sentence sounds a bit strange because grammatically, the subject is missing in this sentence.

Said otherwise, I wonder, what (or who) no longer causes Chrome to crash? Is it NVDA?

Or maybe I haven't understood the sentence correctly...